theano/compile/tests/test_modes.py

```diff
@@ -43,20 +40,14 @@ def test1(self):
         # there should be
         # - VM_Linker
         # - OpWiseCLinker (FAST_RUN)
-        # - WrapLinker ("ProfileMode")
+        # - WrapLinker
         # - PerformLinker (FAST_COMPILE)
         # - DebugMode's Linker  (DEBUG_MODE)
```

Comment:
The comment is now confusing since there are 5 items named, but we assert the list contains 4 elements.  You should probably just remove the line about WrapLinker.